server/portal/libs/agave/operations.py

```diff
@@ -448,20 +447,9 @@ def upload(client, system, path, uploaded_file):
     file_listing = client.files.listFiles(systemId=system, path=path)
     uploaded_file.name = increment_file_name(listing=file_listing, file_name=uploaded_file.name)
 
-    base_url = settings.TAPIS_TENANT_BASEURL
-    token = client.access_token.access_token
     systemId = system
```

Comment:
variable needs to be snake case. You could just use system in line 452.

Reply:
@asimregmi would you mind making this change as well?

Reply:
Made this change. I removed `systemId = system` line as requested. Thank you.